lib/login.dart

```diff
@@ -0,0 +1,216 @@
+          decoration: const BoxDecoration(
+              gradient: LinearGradient(
+                colors:[
+                  Colors.teal,
+                  Colors.white,
+                ],
+                begin: Alignment.topLeft,
+                end: Alignment.bottomLeft,
+                stops:[0.0, 1.0],
+                tileMode: TileMode.clamp,
+              )
+          ),
+        ),
+        title: const Text(
+            "Log in", style:TextStyle(fontSize: 24.0 , color: Colors.black,fontWeight: FontWeight.bold)
+        ),
+      ),
+      body:
+      Container(
+
+        width:double.infinity,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Image.asset('assets/cura-logo.png',height:160.0 ,width:256.0 ),
+            const Text(
+              "Hello! ",
+
+              style: TextStyle(fontWeight: FontWeight.bold, fontSize:16.0 ),
+            ),
+            const Text(
+              "Welcome Back to Cura",
+
+              style: TextStyle(fontWeight: FontWeight.bold, fontSize:16.0 ),
+            ),
+            const SizedBox(
+              height: 25,
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center
+              ,
+              children: [
+                const SizedBox(
+                  height: 25,
+                ),
+                Container(
+                  height:38.0 ,
+                  width: 141.0,
+                  decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(10),
+                      color: Colors.teal
+                  ),
+                  child: MaterialButton(
+                    onPressed: () {},
+                    child:const Text(
+                      "Phone No.",
+                      style: TextStyle(
+                        fontSize:18.0 ,
+                        color: Colors.white,
+                      ),
+                    ),
+                  ),
+                ),
+                Container(
+                  height:38.0 ,
+                  width: 141.0,
+                  decoration: BoxDecoration(
+                      borderRadius: BorderRadius.circular(10),
+                      color: Color(0xFFA2D2D5)
+
+                  ),
+                  child: MaterialButton(
+                    onPressed: () {},
+                    child:const Text(
+                      "Email",
+                      style: TextStyle(
+                        fontSize:18.0 ,
+                        color: Colors.white,
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+
+            ),
+            const SizedBox(
+              height: 25,
+            ),
+
+
+
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.center,
+
+              children: [
+                MaterialButton(
```

Comment:
Please replace MaterialButton with InputText, for both the e-mail and password fields.